packages/bpk-tokens/src/web/base/notifications.json

```diff
@@ -6,7 +6,7 @@
     "category": "notifications"
   },
   "props": {
-    "BANNER_ALERT_BACKGROUND_COLOR": {
+    "BANNER_ALERT_COLOR": {
```

Comment:
As we now use solid colours for the background we don't need the token to be background but instead for text